app/models/user.rb

```diff
@@ -1,6 +1,6 @@
 class User < ActiveRecord::Base
 
-  has_many :questions, inverse_of: :asker
-  has_many :answers,   inverse_of: :answerer
```

Comment:
oh yeah!  Someone who cares about styling.  Kick that unnecessary spacing to the curb